wpilibc/sim/include/WPILib.h

```diff
@@ -28,28 +26,51 @@
 #include "Commands/WaitCommand.h"
 #include "Commands/WaitForChildren.h"
 #include "Commands/WaitUntilCommand.h"
-
-#include "SmartDashboard/SendableChooser.h"
-#include "SmartDashboard/SmartDashboard.h"
-
-#include "AnalogGyro.h"
-#include "AnalogInput.h"
-#include "AnalogPotentiometer.h"
 #include "Counter.h"
 #include "DigitalInput.h"
 #include "DoubleSolenoid.h"
+#include "DriverStation.h"
 #include "Encoder.h"
+#include "ErrorBase.h"
+#include "Filters/LinearDigitalFilter.h"
 #include "GenericHID.h"
 #include "IterativeRobot.h"
 #include "Jaguar.h"
 #include "Joystick.h"
-#include "LiveWindow/LiveWindow.h"
+#include "Notifier.h"
 #include "PIDController.h"
+#include "PIDOutput.h"
+#include "PIDSource.h"
+#include "PWM.h"
+#include "PWMSpeedController.h"
+#include "Relay.h"
+#include "Resource.h"
 #include "RobotBase.h"
 #include "RobotDrive.h"
+#include "SD540.h"
+#include "SPI.h"
 #include "SampleRobot.h"
+#include "SmartDashboard/SendableChooser.h"
+#include "SmartDashboard/SmartDashboard.h"
 #include "Solenoid.h"
+#include "Spark.h"
 #include "SpeedController.h"
 #include "Talon.h"
+#include "Task.h"
+#include "Timer.h"
+#include "Utility.h"
 #include "Victor.h"
+#include "VictorSP.h"
+#include "Vision/AxisCamera.h"
```

Comment:
ehh I'm thinking about the Vision stuff though. Like if you use AxisCamera methods, will you get linker errors? Are those methods defined in athena or shared?